concourse/pipeline-release.yml

```diff
@@ -130,6 +128,15 @@ resources:
       bucket: {{tf-bucket-name}}
       bucket_path: clusters-google/
 
+- name: terraform.d
+  source:
+    access_key_id: ((bucket-access-key-id))
+    bucket: ccp-terraform-provider-plugins
+    region_name: ((aws-region))
+    secret_access_key: ((bucket-secret-access-key))
+    versioned_file: plugin-cache.tgz
+  type: s3
```

Comment:
this one is still on s3?

Reply:
This is OK. in the `gp-concourse-cluster-provisioner` examples, the `terraform.d` resource is still using s3.